department-of-veterans-affairs/vets-api
# rake task for migration itf_datetime to form_start_date on saved_claims

## Summary

- *This work is behind a feature toggle (flipper): NO*
- This rake task is a data migration for moving `itf_datetime` values to `form_start_date` on `saved_claims`. This data migration is necessary before we ultimately remove the `itf_datetime` field. 
- pension benefits team to own this work.

## Related issue(s)

- [*Link to ticket created in va.gov-team repo*](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95626)

## Testing done

- Ran `rake data_migration:saved_claims_form_start_date_update` locally and recorded the result in screenshot below.
- Test plan for rolling out feature:
  - Run rake task in staging and confirm there are no issues. Confirm results with `SavedClaim.where.not(itf_datetime: nil)`. Should return 0 results.
  - Run rake task in production and confirm results with above query.
  - Submit new PRs to remove the rake task and a migration to remove the `itf_datetime` field from the `saved_claims` table.

## Screenshots
![image](https://github.com/user-attachments/assets/90d771aa-aabb-4bb1-a876-9438d58af19f)


## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
